backend/src/config/index.ts

```diff
@@ -54,6 +56,15 @@ const loginServer = {
   LOGIN_SERVER_KEY: process.env.LOGIN_SERVER_KEY || 'a51ef8ac7ef1abf162fb7a65261acd7a',
 }
 
+const blockchainConnector = {
+  BLOCKCHAIN_CONNECTOR_API_URL:
+    process.env.BLOCKCHAIN_CONNECTOR_API_URL || 'http://blockchain-connector:1271',
```

Comment:
Not sure what this means?
Do we have on local a blockchain-connector:1271 or is this a URL that was registered and port 1271 is open for the tests?

Reply:
It will be local, but not yet. I haven't written the Docker and Docker-Compose things yet for that.